cpp/src/io/parquet/parquet_gpu.hpp

```diff
@@ -205,6 +207,7 @@ struct ColumnChunkDesc {
   int8_t codec;                               // compressed codec enum
   int8_t converted_type;                      // converted type enum
   int8_t decimal_scale;                       // decimal scale pow(10, -decimal_scale)
+  bool strict_decimal_type;                   // read decimal data strictly as decimal type
```

Comment:
I think most of this can go away if we take the suggestion to do this all the time.

Reply:
Yes, I've removed it.